exdb/templates/exdb/search_results.html

```diff
@@ -0,0 +1,62 @@
+{% extends "exdb/base.html" %}
+{% load i18n %}
+{% load exdbtags %}
+{% load staticfiles %}
+{% block css %}
+    {{ block.super }}
+    <link rel="stylesheet" type="text/css" href="{% static 'exdb/css/search_results.css' %}" />
+{% endblock %}
+{% block javascript %}
+    {{ block.super }}
+    <script type="text/javascript" src="{% static 'exdb/js/search_results.js' %}"></script>
+{% endblock %}
+{% block content %}
+    {{ block.super }}
+
+    <h1>{% trans "Search Results" %}</h1>
+    <p><a href="{% url 'home' %}">{% trans "Home" %}</a></p>
+
+    {% if experiences %}
+        <table id="search-results" class="tablesorter">
+            <thead>
+                <tr>
+                    <th>{% trans "Experience Name" %}</th>
+                    <th>{% trans "Planners" %}</th>
+                    <th>{% trans "Status" %}</th>
+                    <th>{% trans "Building" %}</th>
+                    <th>{% trans "Keywords" %}</th>
+                    <th data-sorter="shortDate">{% trans "Start Time" %}</th>
+                    <th data-sorter="shortDate">{% trans "End Time" %}</th>
+                </tr>
+            </thead>
+            <tbody>
+                {% for exp in experiences %}
+                    <tr>
+                        <td><a href="{% experience_url exp user %}">{{ exp.name }}</a></td>
+                        <td>
+                            {% for planner in exp.planners.all %}
+                                {{ planner }}{% if not forloop.last %}, {% endif %}
+                            {% endfor %}
+                        </td>
+                        <td>{{ exp.get_status_display }}</td>
+                        <td>
+                            {% for rec in exp.recognition.all %}
+                                {{ rec.affiliation }}{% if not forloop.last %}, {% endif %}
+                            {% endfor %}
+                        </td>
+                        <td>
+                            {% for keyword in exp.keywords.all %}
+                                {{ keyword }}{% if not forloop.last %}, {% endif %}
+                            {% endfor %}
+                        </td>
+                        <td data-text="{{ exp.start_datetime|date:'Y/m/d H:i:s' }}">{{ exp.start_datetime }}</td>
+                        <td data-text="{{ exp.end_datetime|date:'Y/m/d H:i:s' }}">{{ exp.end_datetime }}</td>
```

Comment:
Unfortunately, the text filter at the top of the column only searches the `data-text` attribute instead of the displayed text. We might have to setup a custom textExtractor/parser after all.